autopts/ptsprojects/zephyr/pbp_wid.py

```diff
@@ -0,0 +1,27 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2025, Nordic Semiconductor ASA.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+
+import logging
+
+from autopts.pybtp.types import WIDParams
+from autopts.wid import generic_wid_hdl
+
+log = logging.debug
+
+
+def pbp_wid_hdl(wid, description, test_case_name):
+    log(f'{pbp_wid_hdl.__name__}, {wid}, {description}, {test_case_name}')
```

Comment:
```suggestion
def pbp_wid_hdl(wid, description, test_case_name):
    logging.debug(f'{pbp_wid_hdl.__name__}, {wid}, {description}, {test_case_name}')
```

Reply:
This is part of code which was autogenerated. I'd rather keep it as the other modules.

Reply:
Fair point :)